_dashboards/sm-dashboards.md

```diff
@@ -1,11 +1,12 @@
 ---
 layout: default
-title: Snapshot management
-parent: Index and snapshot management in OpenSearch Dashboards
-nav_order: 17
+title: Snapshot management in Dashboards
+nav_order: 90
+redirect_from:
+  - /dashboards/admin-ui-index/sm-dashboards/
 ---
 
-# Snapshot management
+# Snapshot management in Dashboards
```

Comment:
Now that we have everything under Dashboards, I think we can remove "in Dashboards" from Index management and Snapshot management